src/app/server/OnboardingCodesUtil.cpp

```diff
@@ -64,6 +64,15 @@ void PrintOnboardingCodes(chip::RendezvousInformationFlags aRendezvousFlags)
     {
         ChipLogError(AppServer, "Getting manual pairing code failed!");
     }
+
+    if (GetLongManualPairingCode(manualPairingCode, aRendezvousFlags) == CHIP_NO_ERROR)
```

Comment:
Here, if ` chip::CommissioningFlow::kCustom` is already set in the payload then will this end up print the long pairing code in both the prints:
`ChipLogProgress(AppServer, "Manual pairing code: [%s]", manualPairingCode.c_str());`
`ChipLogProgress(AppServer, "Long Manual pairing code: [%s]", manualPairingCode.c_str());`

Reply:
yes

Reply:
No, we don’t deal with payload in PrintOnboardingCodes(chip::RendezvousInformationFlags aRendezvousFlags) function.